spec/requests/provider_management/external_users_spec.rb

```diff
@@ -25,11 +25,20 @@
         let(:external_user) { create :external_user, user: create(:user, email: 'bubbletea@example.com') }
 
         it 'does not mark the users email as deleted' do
-          pending 'wip'
           expect{ disable_user }.not_to change{ external_user.reload.email }.from('bubbletea@example.com')
         end
       end
 
+      context 'when external user is already deleted' do
+        before do
+          external_user.soft_delete
+        end
+
+        it 'does not mark the users email as deleted a second time' do
+          expect{ disable_user }.not_to change{ external_user.reload.email }.from(/bubbletea@example\.com\.deleted\.\d+$/)
```

Comment:
Is it possible to expect a string `"bubbletea@example.com"` instead of of this regex

Reply:
👍